supertokens_python/recipe/thirdpartypasswordless/api/implementation.py

```diff
@@ -103,6 +103,8 @@ async def create_code_post(self,
                                phone_number: Union[str, None],
                                api_options: PasswordlessAPIOptions,
                                user_context: Dict[str, Any]) -> Union[CreateCodePostOkResult, CreateCodePostGeneralError]:
+        # TODO: Use email delivery ingredient
+        # return await api_options.email_delivery.ingredient_interface_impl.send_email()
```

Comment:
Why is this TODO there? I don't think any api code in this recipe needs changing.

Reply:
I agree. Removed.